doc/source/dev/development_environment.rst

```diff
@@ -249,11 +266,20 @@ Now, you can run::
 
     $ spin gdb mytest.py
 
+In case you are using clang toolchain::
+
+    $ lldb python mytest.py
```

Comment:
I created https://github.com/scientific-python/spin/issues/125 so upstream doesn't lose track that a `spin lldb` would be nice.